Multisig/UI/Safe Management/Add Safe/SelectNetworkViewController/SelectNetworkViewController.swift

```diff
@@ -184,8 +184,10 @@ class SelectNetworkViewController: LoadableViewController, UITableViewDelegate,
         tableView.deselectRow(at: indexPath, animated: true)
         let vc = EnterSafeAddressViewController()
         vc.completion = completion
-        vc.network = model.models[indexPath.row]
-        show(vc, sender: self)
+        vc.network = Network.createOrUpdate(model.models[indexPath.row])
```

Comment:
If add safe flow were canceled, this will introduce new network stored in the database with no safes, which we want to avoid

Reply:
+1